Docker/Dockerfile

```diff
@@ -1,32 +1,29 @@
-############################################################
-# Dockerfile to build OpenZWave Library container images
-# Based on CentOS7
-############################################################
-
-# Set the base image to Ubuntu
 FROM fishwaldo/qt-staticbuilds:5.12.4 as builder
 
-# File Author / Maintainer
 MAINTAINER Justin Hammond
 
-WORKDIR /opt
+RUN dnf -y update \
+ && dnf install -y rapidjson-devel \
+ && dnf clean all
 
-RUN dnf -y install rapidjson-devel
+WORKDIR /src
 RUN git clone https://github.com/qt/qtmqtt.git && cd qtmqtt && git checkout 5.12 && /opt/Qt/5.12.4/bin/qmake && make && make install
 RUN git clone https://github.com/OpenZWave/open-zwave.git && cd open-zwave && make -j4 && make install
 RUN git clone https://github.com/OpenZWave/qt-openzwave.git && cd qt-openzwave && git checkout mqtt && /opt/Qt/5.12.4/bin/qmake && make -j4 && make install
-#RUN ls -lah /usr/local/lib64/
-#RUN ls /opt/
-#RUN ls -lah /ozwdaemon/bin/
-#RUN LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:$LD_LIBRARY_PATH" /ozwdaemon/bin/ozwdaemon
-RUN mkdir /opt/ozw/ && mkdir /opt/ozw/config/
 
-ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:$LD_LIBRARY_PATH"
+FROM fedora:25
```

Comment:
Picked this since it's the base image of fishwaldo/qt-staticbuilds:5.12.4.

Fedora 25 is obsolete. Any plans to update? Next supported version is 26.